discord/state.py

```diff
@@ -1785,6 +1785,28 @@ def parse_voice_server_update(self, data) -> None:
                 )
             )
 
+    def parse_voice_channel_status_update(self, data) -> None:
+        guild = self._get_guild(int(data["guild_id"]))
+        channel_id = int(data["id"])
+        if guild is not None:
+            channel = guild.get_channel(channel_id)
+            if channel is not None:
+                old_status = channel.status
+                channel.status = data.get("status", None)
+                self.dispatch(
+                    "voice_channel_status_update", channel, old_status, channel.status
```

Comment:
Consider adding a `raw_voice_channel_status_update` event

Reply:
yeah, big bots want it